### PR TITLE
add governance manual v2

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Adds v0.2 of Governance Manual.

The consolidated changelog of these updates includes:

- Removal of the Temperature Check step from the Governance Process
- Introduction of a **formal proposal numbering scheme** (e.g. RGP)
- Introduction of **proposal types** (executable, social, consensus)
- Introduction of **monthly proposal cycles**
- Transition of governance process from Discourse to GitHub (see [contributing.md](https://github.com/radicle-dev/radicle-governance/pull/3))

The reasoning behind the proposed changes:

1. In theory, **Temperature Checks** are meant to be informal discussions around an initial idea or question, with the goal of developing a potential proposal. Yet in practice, we found proposers submitting detailed proposal drafts, typically expected in the Discussion step. While Temperature Checks will remain as an informal label, it is removed as a required step in the governance process.  
2. The introduction of a **Formal proposal numbering scheme** and **proposal types** will make keeping track of proposals a lot easier for contributors. Example: `[Executeable][RGP - #] - [PROPOSAL TITLE]`The numbering of proposals throughout their journey of the governance process will make it easier to keep track of multiple active proposals at one time rather than remembering the name. The explicit mention of the proposal type in the proposal title will allow voters to know if the proposals requires an on-chain action or just needs off-chain social consensus. 
3. **Monthly proposal cycles** will sync voting periods and establish a manageable cadence for governance participants. It will also make the proposal process more straight forward and reliable for proposal authors.
4. **Version-controlled documentation** of the governance process will be *crucial to track and learn from changes over time*. We also have included a clear process for updating the governance process, which had not existed before.

Signed-off-by: abbey titcomb <abbeytitcomb@gmail.com>

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -1,166 +1,139 @@
...
 
-- **Functional description** (what is this being proposed?)
-- **Purpose** (what’s the “why”?)
-- **Background** (what is the reasoning behind the proposal?)
-- **Link to Temperature Check**
-- **Reasoning & analysis** (what is the case for the proposal? what are the pros and cons?)
-- **Technical implementation** (who will be writing the code? what is the scope required?)
-- **Impact** (how does this contribute to the long-term resilience, sustainability and/or growth of the Radicle network?)
-- **Open questions** (what else needs to be figured out?)
+A Radicle Governance Proposal (RGP) can be proposed on-chain by anyone who has an amount of $RAD greater than or **equal to 1% of total total supply delegated** to their address. RGPs must be formally labeled in the following format: e.g. `RGP-1`
```
Comment (shelbeeee) [inline manual.md]: Add full example of what a proposal title should look like i.e. include proposal type, # and title: [Executeable][RGP - #] - [PROPOSAL TITLE]

Comment (abbey-titcomb) [reply manual.md]: added in https://github.com/radicle-dev/radicle-governance/pull/2/commits/64fd6363069b0acac9947b6157baaba7a09b4fb2

File: manual.md [hunk 2]
```diff
@@ -1,166 +1,125 @@
...
-- **Impact** (how does this contribute to the long-term resilience, sustainability and/or growth of the Radicle network?)
-- **Open questions** (what else needs to be figured out?)
+A RAD Governance Proposal (RGP) can be proposed on-chain by anyone who has an amount of $RAD greater than or **equal to 1% of total total supply delegated** to their address. RGPs must be formally labeled with the proposal type, #, and title (e.g. [Executeable/Social/Consensus][RGP - #] - [PROPOSAL TITLE])
 
-:::note
+To submit an EGP on-chain, it must first complete **an off-chain review process**. Having a public off-chain review process allows for proposals to be reviewed, discussed, and improved by the Radicle community before reaching an on-chain vote. It also provides on-chain voters with more context on community sentiment and proposal development which ideally leads to more informed and less contentious decisions. Ultimately, involving the community in the public review and development of the proposal is also a way to protect the RadicleDAO, ensuring that what makes its way to an on-chain vote has already been vetted and reviewed in a transparent manner.
```
Comment (shelbeeee) [inline manual.md]: EGP change to RGP!

**Review by shelbeeee — CHANGES_REQUESTED** [review]
See one comment to fix EGP > RGP

File: manual.md [hunk 3]
```diff
@@ -1,166 +1,125 @@
-# RadicleDAO Governance Process
+# Governance Manual [v0.2]
 
-Radicle’s Ethereum integration is maintained and upgraded through a community governance system in which RAD
-token-holders and their delegates debate, propose, and vote on all changes to the network’s Ethereum-based protocols. To
-propose changes, community members must create a Governance Proposal, executable code that outlines a proposed set of
-simple or complex actions and is subject to on-chain voting. Anybody with 1% of RAD delegated to their address can
-propose a Governance Proposal.
+> ❗ This document describes the current governance processes of the RadicleDAO. It is version-controlled and can only updated via off-chain approval. It’s currently maintained [here](https://github.com/ethereum-optimism/Operating-manual) on Radicle.
```
Comment (joelhans) [inline manual.md]: I believe the link here should be updated, unless this is intentional!

File: manual.md [hunk 4]
```diff
@@ -1,166 +1,125 @@
...
-> this process should be considered an active work in progress and is subject to change.
 
-## TL:DR;
+# Overview
 
-Radicle’s governance process is structured around an expectation of self-direction. Community “champions” (the
-individual(s) who created a proposal) are expected to escalate proposals from phase to phase, with guidance from
-community moderators. There are **four stages** in the governance process. To be considered an official Governance
-Proposal, a proposal has to go through all four stages.
+The RadicleDAO is governed by on-chain and off-chain governance processes. These together make up Radicle’s ******************governance system.****************** It’s on-chain governance can be referred to as the DAO’s *********governance module.*********
```
Comment (joelhans) [inline manual.md]: I've noticed that Notion just _loves_ to add these extra asterisks when copy-pasting or converting to Markdown.

```suggestion
The RadicleDAO is governed by on-chain and off-chain governance processes. These together make up Radicle’s **governance system**. It’s on-chain governance can be referred to as the DAO’s **governance module**.
```

File: manual.md [hunk 5]
```diff
@@ -1,166 +1,125 @@
...
+### On-chain
 
-1. Create a **Temperature Check** to initiate an informal discussion around a proposed idea and/or change.
-2. Use feedback from the Temperature Check to draft a proposal that meets the criteria set in this document and start a
-   **Discussion** (t/r 5 days)
-3. Submit the draft proposal for **Formal Review** by creating a [Snapshot poll](https://snapshot.org/#/gov.radicle.eth)
-   (t/r 5 days)
-4. "If passed, deploy a Governance Proposal via [Boardroom](https://boardroom.io/radicle/proposals) or
-   [Tally](https://www.tally.xyz/gov/eip155:1:0x690e775361AD66D1c4A25d89da9fCd639F5198eD). (t/r 3 days)
+The RadicleDAO’s Treasury is controlled by a tokenized governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token ($RAD). As members of the RadicleDAO,  $RAD holders participate in the decision-making around the treasury by voting and creating on-chain governance proposals composed of executable actions.
```
Comment (joelhans) [inline manual.md]: Extra space!

```suggestion
The RadicleDAO’s Treasury is controlled by a tokenized governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token ($RAD). As members of the RadicleDAO, $RAD holders participate in the decision-making around the treasury by voting and creating on-chain governance proposals composed of executable actions. 
```

File: manual.md [hunk 6]
```diff
@@ -1,166 +1,125 @@
...
-  **Temperature Check**.
+Each proposal type has a designated [template](https://www.notion.so/Proposal-Template-Library-e8b3408c8c0a4c2d8288629f183a5f54).
 
-Anybody can escalate a Temperature Check to a Discussion by putting together an draft proposal that meets these
-criteria. If a Discussion is started without demonstrating a successful Temperature Check, then it will be flagged and
-closed by community moderators.
+### Proposal Cycles
 
-### Phase 3: Formal Review :herb: - Discourse + Snapshot
+The RadicleDAO manages proposals in ****************************monthly cycles**************************** to sync voting periods and establish a manageable cadence for governance participants.
```
Comment (joelhans) [inline manual.md]:
```suggestion
The RadicleDAO manages proposals in **monthly cycles** to sync voting periods and establish a manageable cadence for governance participants. 
```

File: manual.md [hunk 7]
```diff
@@ -1,166 +1,125 @@
...
-closed by community moderators.
+### Proposal Cycles
 
-### Phase 3: Formal Review :herb: - Discourse + Snapshot
+The RadicleDAO manages proposals in ****************************monthly cycles**************************** to sync voting periods and establish a manageable cadence for governance participants. 
 
-After a proposal is thoroughly discussed, the proposal can be submitted to the community for **Formal Review**.
+| Monthly Schedule | Stage | Weekly Schedule | Action |
+| --- | --- | --- | --- |
+| Week 1 | Discussion | First Monday  | Post proposal to forum for Discussion* |
```
Comment (joelhans) [inline manual.md]:
```suggestion
| Week 1 | Discussion | First Monday  | Post proposal to forum for Discussion |
```

File: manual.md [hunk 8]
```diff
@@ -1,166 +1,125 @@
...
+| Week 2 | Discussion | Second Wednesday | Join monthly Governance Call to present and discuss proposal with community |
+| Week 3 | Formal Review | Third Monday |  Open proposal for Formal Review |
+| Week 4 | Submission | Fourth Monday | Submit proposal on-chain |
 
-To do this:
+**Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but requires at least 7 days on the forum to be considered for that month’s voting cycle*
```
Comment (joelhans) [inline manual.md]:
```suggestion
**Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but requires at least 7 days on the forum to be considered for that month’s voting cycle.**
```

File: manual.md [hunk 9]
```diff
@@ -1,166 +1,125 @@
...
 
-- Create a Snapshot poll that includes the updated overview of the proposal and options on how to move forward. The
-  options can be multiple choice (if to present multiple options for the proposal) or be binary, but must include the
-  choice Make no change. **Voting for Snapshot polls should start on a Monday, at 5pm CET and end on the following
-  Monday, at 5pm CET**.
+### Voting Thresholds
 
-  :::note
+Once proposed, participants can vote for/against the proposal on-chain with their RAD. An RGP is ****************approved**************** if it satisfies the following minimum vote thresholds:
```
Comment (joelhans) [inline manual.md]:
```suggestion
Once proposed, participants can vote for/against the proposal on-chain with their RAD. An RGP is **approved** if it satisfies the following minimum vote thresholds:
```

File: manual.md [hunk 10]
```diff
@@ -1,166 +1,125 @@
...
 
-  You can’t edit a Snapshot poll after it is submitted, so please make sure you include as much information as possible.
+- ****************************Quorum:**************************** The minimum number of total RAD votes required to be cast in connection with a proposal. Quorum is currently measured as *********************************a % of the total RAD supply.********************************* The required quorum to pass a proposal is **4% (4M RAD) of total supply (100M RAD)**
```
Comment (joelhans) [inline manual.md]:
```suggestion
- **Quorum:** The minimum number of total RAD votes required to be cast in connection with a proposal. Quorum is currently measured as **a % of the total RAD supply.** The required quorum to pass a proposal is **4% (4M RAD) of total supply (100M RAD)**
```

File: manual.md [hunk 11]
```diff
@@ -1,166 +1,125 @@
...
+    
+    *******************Formal Reviews******************* and *Submissions* are required to meet quorum requirements.
```
Comment (joelhans) [inline manual.md]:
```suggestion
    **Formal Reviews** and *Submissions* are required to meet quorum requirements.
```

File: manual.md [hunk 12]
```diff
@@ -1,166 +1,125 @@
...
+    
+- ********Approval Threshold:******** If quorum is met, the proposal passes if a majority of the votes are “in-favor”.
```
Comment (joelhans) [inline manual.md]:
```suggestion
- **Approval Threshold:** If quorum is met, the proposal passes if a majority of the votes are “in-favor”.
```

File: manual.md [hunk 13]
```diff
@@ -1,166 +1,125 @@
...
 
-  :::
+**Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but requires at least 7 days on the forum to be considered for that month’s voting cycle*
 
-- Create a topic and label it as follows: “Formal Review :herb: - Your Title Here”. Link the Snapshot poll in the topic. Any topic that is labeled Formal Review and does not link to a successful Temperature check and/or does not meet the criteria outlined above will be flagged and removed.
-- All Radicle stakeholders are responsible for ensuring proper review of any proposal in Formal Review. This includes the core team. However, as the ‘Champion’ of the proposal, it is your responsibility to gather support for your proposal by sourcing delegates, actively responding to questions, and addressing feedback regarding the proposal.
+*****Submission triggers a 3-day voting period. If passed, the proposal is queued for 48hrs, then it can be executed.***
```
Comment (joelhans) [inline manual.md]:
```suggestion
**Submission triggers a 3-day voting period. If passed, the proposal is queued for 48hrs, then it can be executed.** 
```

File: manual.md [hunk 14]
```diff
@@ -1,166 +1,125 @@
...
-- All Radicle stakeholders are responsible for ensuring proper review of any proposal in Formal Review. This includes the core team. However, as the ‘Champion’ of the proposal, it is your responsibility to gather support for your proposal by sourcing delegates, actively responding to questions, and addressing feedback regarding the proposal.
+*****Submission triggers a 3-day voting period. If passed, the proposal is queued for 48hrs, then it can be executed.*** 
 
-At the end of 5 days, whichever option has the majority of votes wins. 4% of participation is required for a Snapshot to pass Formal Review. Only then should it be deployed as an official Governance Proposal. If the option “Make no change” wins, the topic should be archived by community moderators.
+## Voting
 
-### Phase 4: Governance Proposal :sunflower:
+Voting requires Radicle (RAD) tokens, which you can [obtain here](https://docs.radicle.xyz/docs/connecting-to-ethereum/obtaining-rad). One RAD token is equal to one vote. Voting takes place both on- and off-chain:
```
Comment (joelhans) [inline manual.md]: New URL for this page!

```suggestion
Voting requires Radicle (RAD) tokens, which you can [obtain here](https://docs.radicle.community/governance/obtain-rad). One RAD token is equal to one vote. Voting takes place both on- and off-chain:
```

File: manual.md [hunk 15]
```diff
@@ -1,166 +1,125 @@
...
 
-1. Write the proposal code and deploy the proposal via [rad gov](https://github.com/radicle-dev/radgov). All proposed
-   code should be audited by a professional auditor.
-2. Ensure at least 1 million $RAD is delegated to your address. If you do not have enough RAD to create a proposal, find
-   a delegate to either delegate to you or propose on your behalf.
-3. Create a topic titled “Governance Proposal [Proposal Number] — [Your Title Here]”. Be sure to include:
-   - A link to the official proposal in gov.radicle.network.
-   - Links to any relevant Snapshot polls/discussion threads and link to any relevant Snapshot polls/discussion threads.
-   - A full overview of the proposal, with any feedback or changes introduced during the Formal Review.
+Once proposed, participants can vote for/against the proposal on-chain with their RAD. An RGP is ****************approved**************** if it satisfies the following minimum vote thresholds:
```
Comment (joelhans) [inline manual.md]:
```suggestion
Once proposed, participants can vote for/against the proposal on-chain with their RAD. An RGP is **approved** if it satisfies the following minimum vote thresholds:
```

File: manual.md [hunk 16]
```diff
@@ -1,166 +1,125 @@
...
 
-*Topics that begin with “Governance Proposal” that have not successfully passed a Temperature Check and a Formal Review
-should be removed by community moderators.*
+- ****************************Quorum:**************************** The minimum number of total RAD votes required to be cast in connection with a proposal. Quorum is currently measured as *********************************a % of the total RAD supply.********************************* The required quorum to pass a proposal is **4% (4M RAD) of total supply (100M RAD)**
```
Comment (joelhans) [inline manual.md]:
```suggestion
- **Quorum:** The minimum number of total RAD votes required to be cast in connection with a proposal. Quorum is currently measured as **a % of the total RAD supply.** The required quorum to pass a proposal is **4% (4M RAD) of total supply (100M RAD)**
```

**Review by joelhans — COMMENTED** [review]
Hey! Hope you don't mind me jumping into the Markdown and making a few suggestions. I don't know if any of these are _necessary_, except for the new URL for obtaining $RAD, but I thought it would be better to chime in before this is merged, which would then require a follow-up process.

**Comment by abbey-titcomb** [comment]
@joelhans - just resolved all your comments with my latest commit. thanks for the review!

**Review by abbey-titcomb — COMMENTED** [review]
fixed all comments with my latest commit

File: manual.md [hunk 17]
```diff
@@ -1,166 +1,125 @@
...
+| Type | Description | Process |
+| --- | --- | --- |
+| Executable | On-chain actions such as treasury distributions, updates to contract parameters, and upgrades to governance system. | Discussion → Formal Review → Submission |
+| Social | Changes that can not be enforced by the DAO on-chain (e.g. updates to Governance Manual) | Discussion → Formal Review  |
+| Consensus | Ad-hoc changes that “should” have community approval, but aren’t classified by an Executable or Social proposal | Discussion → Formal Review |
 
-- Create a topic and label it as follows: “Discussion :seedling: - Your Title Here”.
-- Include an overview of the drafted proposal that meets the criteria outlined above and link to it’s previous
-  **Temperature Check**.
+Each proposal type has a designated [template](https://www.notion.so/Proposal-Template-Library-e8b3408c8c0a4c2d8288629f183a5f54).
```
Comment (shelbeeee) [inline manual.md]: @abbey-titcomb shouldn't this link to the Template Library PR4 no the Notion page?

**Comment by abbey-titcomb** [comment]
made all outstanding changes!

**Review by shelbeeee — APPROVED** [review]
Good to go!
